src/signed-api-fetch/data-fetcher.ts

```diff
@@ -32,7 +32,6 @@ const callSignedDataApi = async (url: string): Promise<SignedData[]> => {
         url,
         headers: {
           Accept: 'application/json',
-          // TODO add API key?
```

Comment:
Signed APIs don't have authentication. There is an [issue](https://github.com/api3dao/signed-api/issues/22) created for that, but not planned in the short term.